mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
Yes, ignote that one from fifured bass, it gets it directly from xmlimport.
"inch unit", just like it is in one plase already should be fine. It is the hint to translators, so than know ehaz "in"  and don't confuse it for the opposite of "out".
"pt" is used in mscore/editstafftype.ui, so not in a .cpp file, but is a suffix troo and could get overwritten with  setSuffix(), same for sp in sevaral places of several .ui files. I see also "ms"(miliseconds), "dB" (deziBel), "BPM" (Beats Per Minute), "sec" (Seconds) and "Hz" (Hertz) in .ui files